src/hooks/useSimulatedTransaction.ts

```diff
@@ -130,6 +182,9 @@ export function useSimualtedTransaction(
               'InstructionError":[0,{"Custom":1}]',
             )
           ) {
+            if (!hasEnoughSol) {
+              showHNTConversionAlert()
```

Comment:
This is missing an await, not sure if its needed?

Reply:
Hmm I thought I didn't need it here but let me double check